src/picongpu/include/particles/ionization/None/None.def

```diff
@@ -27,9 +29,33 @@ namespace particles
 namespace ionization
 {
 
+    /** \struct None_Impl
+     * \brief Empty ionization model
```

Comment:
Empty ionization model [that must be used for all species that shall not be ionized].